docs/test.py

```diff
@@ -40,7 +41,31 @@ def test_rtd_pydoctor_multiple_call():
 
 def test_rtd_extension_inventory():
     """
-    The Sphinx inventory is available during normalsphinx-build.
+    The Sphinx inventory is available during normals sphinx-build.
     """
-    with open(BASE_DIR / 'usage.html', 'rb') as stream:
-        assert 'href="/en/latest/api/pydoctor.sphinx_ext.build_apidocs.html"' in stream.read().decode()
+    with open(BASE_DIR / 'usage.html', 'r') as stream:
+        data = stream.read()
+        assert 'href="/en/latest/api/pydoctor.sphinx_ext.build_apidocs.html"' in data, data
+
+
+def test_sphinx_object_inventory_version():
+    """
+    The Sphinx inventory is generated with the project version in the header.
+    """
+    # The pydoctor own invetory.
+    with open(BASE_DIR / 'objects.inv', 'rb') as stream:
+        data = stream.read()
+        assert data.startswith(
+            b'# Sphinx inventory version 2\n'
+            b'# Project: pydoctor\n'
+            b'# Version: ' + __version__.encode() + b'\n'
+            ), data
+
+    # The demo/showcase inventor for which we have a fixed version.
+    with open(BASE_DIR / 'docformat' / 'epytext' / 'demo' / 'objects.inv', 'rb') as stream:
```

Comment:
This could be put in a separate test case, so we'll have more information when the first checks fails.

Reply:
Was moved to separate test.